Add ability to scan for hashes of vulnerable .class files

gmoniker opened the pull request:
See my answer on https://serverfault.com/a/1086290/188737

This is a bit roughshod, did a minimal test with a couple of jar files. File vulnerable.hashes with sha256 hashes in hex format, ordered, one per line has to be provided.

gmoniker committed:
Update log4j_checker_beta.sh

gmoniker committed:
Update log4j_checker_beta.sh

Add vulnerable hashes checker for .class files inside java archives. Assumes a vulnerable.hashes file in current working directory with ordered sha256 hashes in hex, one per line.

rubo77 commented:
Could you please add three things:
- an automatic download of the `./vulnerable.hashes`
- an OK message if nothing is found
- a line in the README that explains this feature

gmoniker committed:
Update log4j_checker_beta.sh

Add the feature of specifying a plain text download of a file with sha256 checksums of vulnerable .class files. The present routine will only work on first level of archives.

gmoniker committed:
Update README.md

gmoniker commented:
I have added the option of downloading a checksum file.  I did not include an actual URL, this exercise is left to the reader, as they say. And added the OK message and line in the README.

rubo77 commented:
> I did not include an actual URL, this exercise is left to the reader, as they say

Why did you not include a URL? I didn't find anythint where "they say" we shouldnt?
